services/slack/parse.go

```diff
@@ -198,6 +198,12 @@ func (t *Transformer) SlackConvertUserMentions(users []SlackUser, posts map[stri
 			for mention, r := range regexes {
 				post.Text = r.ReplaceAllString(post.Text, mention)
 				posts[channelName][postIdx] = post
+
+				if post.Attachments != nil {
+					for _, attachment := range post.Attachments {
+						attachment.Fallback = r.ReplaceAllString(attachment.Fallback, mention)
```

Comment:
Not sure I understand this line. Doesn't this effectively do the following?

```suggestion
						attachment.Fallback = mention
```

Reply:
I understand now. I didn't realize `r` was its own regex